tools/cpp/cc_configure.bzl

```diff
@@ -169,6 +169,7 @@ cc_autoconf = repository_rule(
         "GCOV",
         "HOMEBREW_RUBY_PATH",
         "SYSTEMROOT",
+        "USER",
```

Comment:
I'm not sure `USER` has to be here, as far as I could tell changing `USER` would force this even without adding it here, so maybe it supersedes this list, but it seems like semantically this is still the right thing to do.

Reply:
Let's still add it then.